plugins/ingest/src/test/resources/rest-api-spec/test/ingest/80_simulate.yaml

```diff
@@ -0,0 +1,139 @@
+---
```

Comment:
In process of adding tests for when exceptions are thrown during pipeline/processor execution